cmd/conf/chain.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 type L1Config struct {
```

Comment:
do we want to rename L1Config / L1ConfigAddOption / default / etc as well?

Reply:
Yes, but I figured we should do that after the execution split to avoid merge conflicts. I also wanted to make this PR as small as possible to review.